rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/dto/JobResultData.java

```diff
@@ -83,9 +87,26 @@ public void setExceptionResults(Map<String, TaskResultData> exceptionResults) {
         this.exceptionResults = exceptionResults;
     }
 
+    public void setResultMap(Map<String, String> resultMap) {
+        this.resultMap = resultMap;
+    }
+
+    public Map<String, String> getResultMap() {
+        return resultMap;
+    }
+
+    public Map<String, String> getSerializedResultMap() {
+        return serializedResultMap;
+    }
+
+    public void setSerializedResultMap(Map<String, String> serializedResultMap) {
+        this.serializedResultMap = serializedResultMap;
```

Comment:
Can you use lombok libraries and annotations instead of writing getters and setters ?

Reply:
lombok is not used in this part of the code, introduction of lombok is not part of this pull request.